### ISSUE TITLE
License Concern

### ISSUE THREAD

**Issue by aikar** [TheBusyBiscuit/MagicLoot3#1]
Was linked this project by someone, but noticed you're using CC Non-Commercial.

This isn't really a great license for software, and it's easy to argue that almost every Minecraft Server is commercial. (The only way you can classify as Non-Commercial is if you do not take any form of payments nor run any advertisements on your website unless you register as an official non-profit)

I suggest switching to a more common software license, such as GPL, MIT or Apache (I Recommend MIT)

**Comment by TheBusyBiscuit** [comment]
Ah thats why I love the internet...
People pretend to have knowledge they dont have and yell at me for something they didnt even do research on...

No, I wont change the license. The "NonCommercial" part refers to the distribution of the work aka that noone is allowed to sell this Plugin for money...
And next time read the license before pretending to educate someone....

**Comment by TheBusyBiscuit** [comment]
"NonCommercial" should only refer to distributing the work, not using it.
I would change the license to an updated version of it, if youd point me out the exact part of the legal text that states that...

Besides that I quit development months ago, not continuing this project anyway...

**Comment by TheBusyBiscuit** [comment]
**Well, if you want to take over the project, feel free to change the license.**
I just picked a license which prevents anyone from selling my work, besides that english isn't my native language and that legal text can always be interpreted very very vaguely...

As to why I might have sounded quite harsh, I havent slept for two days as Im sick, so Im not really in a good mood right now, I apologise if I sounded harsh.

I have also quit development on all projects several months ago and I still get 20+ messages each day about people asking questions, already spam-filtered most websites except for github. Again, if you want to over the project, feel free to change the license, but I am not gonna bother with this project anymore.
